src/user/system/CaptainManager.ts

```diff
@@ -671,6 +672,93 @@ class CaptainManager {
             })
     }
 
+    updateGoAccessInfo(goAccessInfo: GoAccessInfo) {
```

Comment:
Added it to the CaptainManager.init, is this a good place.  Nothing in the volume should get wiped during service restart since it's a host volume mount right?

Reply:
Yes, volume remains intact, but `generated` subdirectory gets wiped out and regenerated on restart.

And yes, init is the right place.